packages/flutter/lib/src/widgets/sensitive_content.dart

```diff
@@ -0,0 +1,298 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' show max;
+
+import 'package:flutter/services.dart' show ContentSensitivity, SensitiveContentService;
+
+import '../../widgets.dart' show ConnectionState;
```

Comment:
we should not import the widgets.dart directly, instead import the file required in the widget folder directly